### PR TITLE
Bower json

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

**Comment by smgcode** [comment]
LGTM

**Comment by AlecRaeside** [comment]
LGTM

File: bower.json [hunk 1]
```diff
@@ -3,12 +3,16 @@
   "version": "0.0.2-354",
   "homepage": "https://github.com/westfieldlabs/scally",
   "authors": [
-    "Chris Pearce <CPearce@westfield.com>"
+    "Chris Pearce <cpearce@westfield.com>",
+    "Alec Raeside <araeside@westfield.com>",
+    "Kate Levin <klevin@westfield.com>"
```
Comment (kllevin) [inline bower.json:8]: much better :)

**Comment by kllevin** [comment]
LGTM

**Pull request merged**